src/builder/builder.go

```diff
@@ -110,6 +111,7 @@ func (b *Builder) ReadBuilderConf() {
 		{`^SERVER_STATE_DIR\s*=\s*`, &b.Statedir},
 		{`^VERSIONS_PATH\s*=\s*`, &b.Versiondir},
 		{`^YUM_CONF\s*=\s*`, &b.Yumconf},
+		{`^CONTENTURL\s*=\s*`, &b.Contenturl},
```

Comment:
This should be renamed to UPSTREAMURL or something like that, since CONTENTURL already exists in the builder.conf and refers to the location clients running the mixer should update from.

Reply:
@tmarcu Done. New code pushed for review.